src/EnergyPlus/WaterUse.cc

```diff
@@ -1092,117 +1099,142 @@ namespace WaterUse {
         this->TotalMassFlowRate = this->TotalVolFlowRate * Psychrometrics::RhoH2O(DataGlobalConstants::InitConvTemp);
 
         // Calculate hot and cold water mixing at the tap
-        if (this->TotalMassFlowRate > 0.0) {
+        if (this->TotalMassFlowRate > 0.0 && this->allowHotControl) {
```

Comment:
If hot water mixing is allowed, do this mixing, otherwise drop to end and set the else

Reply:
Priority 2: get this logic right

Reply:
It's a bit tough to follow here on the GitHub view, but still, seems reasonable.